DEVREL-187 Box Developer Comminuty info (#294)
box/box-developer-changelog: content/2023/06-13-join-box-developer-community.md

---
applied_at: "2023-06-13"
applies_to: 
- api
- node
- python
- java
- windows
- cli
- ui-elements
- reports
- salesforce
- ios
- android
- sdks
is_impactful: false
is_new_feature: true
collapse: true
show_excerpt: true
release_source_url: ''
---

# Join Box Developer Community 

> 👋 Exciting news for developers and admins! Box has launched the Box Developer Community site, where you can connect with peers and experts to build innovative applications on top of the Box cloud content management and collaboration platform. Join us today at https://forum.box.com

Reviewer: ```suggestion
👋 Exciting news for developers and admins! Box has launched the Box Developer Community site, where you can connect with peers and experts to build innovative applications on top of the Box cloud content management and collaboration platform. Join us today at https://forum.box.com
```
```suggestion
👋 Exciting news for developers and admins! Box has launched the Box Developer Community site, where you can connect with peers and experts to build innovative applications on top of Box cloud content management and collaboration platform. Join us today at https://forum.box.com!
```

I would add something at the end :D

<!-- more -->

# Join, connect and learn

> * Get support & provide feedback

Reviewer: I would either use `and` in all bullet points or `&` to make it consistent :)

> * Share and become a community leader

Reviewer: It seems like something is missing after `Share`. Maybe `Share experiences` or `Share knowledge`?

* Connect and chat with other Box developers and admins 

> * Learn from other community members and Box subject matter experts form Box

Reviewer: ```suggestion
* Learn from other community members and Box subject matter experts
```

> * Stay up to date with newest features, updates, developer-focused events, blog posts, and more!

Reviewer: ```suggestion
* Stay up to date with the newest features, updates, developer-focused events, blog posts, and more!
```

## New place to get support

> From now on you can ask questions and seek for guidance in our new [Box Developer Community forum][1]. In this new space, you’ll be able to speak directly to the Box Developer Relations team, as well as your peers and other subject matter experts from Box.

Reviewer: ```suggestion
From now on you can ask questions and seek for guidance in our new [Box Developer Community forum][1]. Use this new space to  directly speak to the Box Developer Relations team, as well as your peers and other subject matter experts from Box.
```
